packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Button, Modal, DesktopWrapper, MobileDialog, MobileWrapper, UILoader, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { useStores } from 'Stores/index';
+import TradigPlatformIconProps from 'Assets/svgs/trading-platform';
+import { TModalContent, TAccountType, TAccountCard, TTradingPlatformAvailableAccount } from './types';
+import { TIconTypes } from 'Types';
+
+const derived_account: TAccountType = {
+    title_and_type: 'Derived',
+    icon: 'Derived',
+    description: 'Trade CFDs on MT5 with Derived indices that simulate real-world market movements.',
```

Comment:
need localize here

Reply:
I get these values as props and use them in AccountCard component inside localize.
No need to use localize twice